packages/linea-ens-resolver/test/testL1Resolver.spec.ts

```diff
@@ -448,11 +456,42 @@ describe("Crosschain Resolver", () => {
     await rollup.setCurrentStateRoot(currentBlockNo, currentStateRoot);
   });
 
-  it("should revert if the block number returned by the gateway is not the most recent one", async () => {
+  it("should not revert if the block number returned by the gateway is in the accepted block range", async () => {
     const currentBlockNo = await rollup.currentL2BlockNumber();
     const currentStateRoot = await rollup.stateRootHashes(currentBlockNo);
-    // Put a wrong block number
-    await rollup.setCurrentStateRoot(blockNo + 100_000, stateRoot);
+    await rollup.setCurrentStateRoot(blockNo, stateRoot);
+    await rollup.setCurrentStateRoot(
+      blockNo + ACCEPTED_BLOCK_RANGE_LENGTH - 10,
+      stateRoot
+    );
+    let proofsEncoded = AbiCoder.defaultAbiCoder().encode(
+      [
+        "uint256",
+        "tuple(bytes key, uint256 leafIndex, tuple(bytes value, bytes[] proofRelatedNodes) proof)",
+        "tuple(bytes32 key, uint256 leafIndex, tuple(bytes32 value, bytes[] proofRelatedNodes) proof, bool initialized)[]",
+      ],
+      [blockNo, proofTest.accountProof, proofTest.storageProofs]
+    );
+    proofsEncoded = PROOF_ENCODING_PADDING + proofsEncoded.substring(2);
+
+    const result = await target.getStorageSlotsCallback(
+      proofsEncoded,
+      extraDataTest
+    );
+    expect(result).to.not.be.null;
```

Comment:
how do you test the response is good vs. just checking the result is not empty?

Reply:
I'm testing it's not reverting

Reply:
do you have tests checking the values returned are correct? in particular with the extraData with > 32 bytes

Reply:
No I'm adding it now

Reply:
@thedarkjester Actually I'm having troubles to get a result that makes sense to check against when calling directly `getStorageSlotsCallback` in the test cases. 
Would you mind trying on your side ? the data in the response is huge and I don't know what to make of it.

Reply:
I have added predefined values matching what's in the ret [value](https://github.com/Consensys/linea-ens/blob/a7c00fbb783747fa46532ff384e504230209a930/packages/linea-state-verifier/contracts/EVMFetchTarget.sol#L49) to check against since that's the only thing I can make sense of.
Let me know what you think